armi/settings/fwSettings/tightCouplingSettings.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2019 TerraPower, LLC
```

Comment:
Random thought. Should the date on this be updated?

Reply:
I think it's like that everywhere. Yes we should throughout the code base as a different single PR

Reply:
@john-science

Reply:
At one point, I was told that the year in the copyright should be set to the year in which the file was created. So for this file, it should be 2022 or 2023.

Reply:
No, this file should say 2023 on it.

The current year is 2023.  

We should not be claiming our copyright existed years before we wrote the code. That would be lying, and minorly illegal.